.kitchen.yml

```diff
@@ -25,6 +25,7 @@ platforms:
   - name: centos-7
   - name: debian-8
```

Comment:
shall we remove debian-8 support? 
I only really want to do, current-1

Reply:
Yeah I'll pull that out. Gotta fix the Deb 10 tests